internal/preflight/dryrun.go

```diff
@@ -12,25 +15,63 @@ type CreationDryRun struct {
 }
 
 func NewCreationDryRun(client client.Writer) *CreationDryRun {
```

Comment:
Can we rename this object to just DryRun?
Now it's doing both creation and update testing.